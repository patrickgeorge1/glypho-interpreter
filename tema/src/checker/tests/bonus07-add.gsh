ii+1+o
