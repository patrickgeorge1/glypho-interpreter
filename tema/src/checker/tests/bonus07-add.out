FAB8BAB1A2A6FA010AB98AA0A984FFBB98410365AA48BB94120FE1098AAABE41023E303
